blocklib/blocks/copy/copy.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <gnuradio/sync_block.hpp>
+namespace gr {
+namespace blocks {
+
+class copy : public sync_block
+{
+public:
+    typedef struct {
```

Comment:
Tiny thing: this pattern is not necessary in C++. `struct block_args` works just fine. `struct` is just a `class` with default `public`.

Reply:
thanks for catching this - old habits die hard.